css/style.css

```diff
@@ -62,13 +63,41 @@ html
     -moz-transition: -moz-transform 0.6s ease-out;
     -o-transition: -o-transform 0.6s ease-out;
     transition: transform 0.6s ease-out;
-    }
-    .overlay:target+.popup {
+}
+.overlay:target+.popup 
+{
     -webkit-transform: translate(-50%, 0);
     -ms-transform: translate(-50%, 0);
     -o-transform: translate(-50%, 0);
     transform: translate(-50%, 0);
     top: 20%;
-    }
-     
+}
+.login-form 
+{
+	width: 340px;
+    margin: 50px auto;
```

Comment:
There are still a lot of things that you can do through bootstrap that you have in your css files, like margin and width here, read the bootsrap docs(which is going to take a while) or just google if a given class(for example bootsrap padding) is present in the framework.

Reply:
Okay,  I`ll try  read moer about this problem and fix it